feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,179 @@
+# AFT-1.1: AFTs Base
+
+## Summary
+
+IPv4/IPv6 unicast routes next hop group and next hop.
+
+## Testbed
+
+* atedut_12.binding
```

Comment:
This is the most expensive testbed using 12 ports.  I see this test only requires 3 ports?  So please use the 4 port testbed at: 

https://github.com/openconfig/featureprofiles/blob/cedbeabcdbea008ebb070ed6952346ae0178a953/topologies/atedut_4.testbed

Reply:
done